tools/docker/docker-compose.ts-integration.yaml

```diff
@@ -14,7 +14,9 @@ services:
       - /var/run/docker.sock:/var/run/docker.sock
     depends_on:
       - node
+      - node-2
```

Comment:
Ah man, so much duplication, is there anyway we can DRY up our compose configs?

Reply:
Between `integration.yaml` and `ts-integration.yaml`, yeah there is a lot of duplication. I'm a bit reluctant to refactor only because we should be removing `integration.yaml` in the future after we convert the remaining tests to typescript.

Reply:
Fair enough